documentation/release-notes/topics/version_4.md

```diff
@@ -1,5 +1,92 @@
 # Version 4
 
+## Hyperview 4.8 (TBD)
+
+This section covers significant changes and bug fixes in Hyperview 4.8.x since version 4.7.x
+
+:::{important}
+1. This release has changes and improvements to the Data Collector software. We strongly recommend updating your installed Data
+   Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+2. Support for the *Windows* version of the Data Collector has ended on *January 31, 2025*. Customers must switch
+   to the Linux version of the Data Collector. Detailed {ref}`installation instructions <Setting-up-data-collectors-doc>`
+   are in the product documentation.
+
+3. The communication provider for email notification has been changed from Twilio SendGrid to Microsoft Azure Communication
+   Services. Notification emails will come from a **DoNotReply@SUBDOMAIN.azurecomm.net** domain. The SUBDOMAIN
+   will change based on the azure region.
+:::
+
+### Enhanced Feature: Microsoft Teams Notification Channels
+
+Microsoft has [announced](https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/)
+the sudden and quick retirement of the Webhook interface within Microsoft Teams. With this change we have updated the
+functionality to support the new recommended approach.
+
+The functionality will still use webhooks, albeit, the webhook will need to be created using a Workflow instead of a Connector.
+More information is in the [Microsoft documentation](https://support.microsoft.com/en-us/office/create-incoming-webhooks-with-workflows-for-microsoft-teams-8ae491c7-0394-4861-ba59-055e33f75498).
+
+:::{important}
+If you have setup notifications using the deprecated Webhook interface, please convert to the new method to avoid an
+interruption in notifications. Contact Hyperview support if you need any assistance or advice on the matter.
+:::
+
+### Enhanced Feature: Layout editor
+
+1. A new "Multi Select" menu has been added the layout header action item group to allow for group select of various layout components.
+   Capabilities have been added for select all, select all shapes, select all Racks, and select all assets as well as a rectangular
+   selection box that can be drawn on the layout. Once selected items can be removed or group edited depending on context.
+
+2. The view filter for rack total power has been proved to dynamically read the rack design value from the parent location and adjust
+   the color schema based on the setting.
+
+### Enhanced Feature: Rack power widget
+
+To allow for better power planning, The Rack Power Widget has been enhanced to show total template power for devices in the rack and
```

Comment:
"The Rack Power Widget" should be "the Rack Power widget".

Reply:
Fixed, thanks!